website/.prettierignore

```diff
@@ -1,25 +1,17 @@
 # unconventional js
 /blueprints/*/files/
-/vendor/
 
 # compiled output
 /dist/
-/tmp/
-
-# dependencies
-/bower_components/
-/node_modules/
 
 # misc
 /coverage/
 !.*
-.eslintcache
-.lint-todo/
+.*/
+/pnpm-lock.yaml
+ember-cli-update.json
+*.html
+*.scss
 
-# ember-try
-/.node_modules.ember-try/
-/bower.json.ember-try
-/npm-shrinkwrap.json.ember-try
-/package.json.ember-try
-/package-lock.json.ember-try
-/yarn.lock.ember-try
+# temporary ignore
+/docs/
```

Comment:
it might be beneficial to enable this, but when I ran it the diff was huge so I decided to keep it out for now